content/cloud/noops-sql-db.md

```diff
@@ -1,13 +1,11 @@
-title = "NoOps SQL Database"
+title = "NoOps SQL Database Tutorial"
```

Comment:
Do you consider this a "tutorial"?  It's a bit of an odd hybrid, some stuff about how to create a Spin app (?) but then more referency content about using the CLI to create and delete databases.

Reply:
It is a weird hybrid of Tutorial and Reference. I can open an issue to revisit this page with a content refresh so we don't forget it needs a bit of TLC

Reply:
Thanks @macolso this is ok for now, as we have a time limit to synchronize this PR with the announcement of Fermyon Cloud’s Birthday. We can certainly revisit and update, for sure.